lib/include/secw_sshkey_and_login.h

```diff
@@ -0,0 +1,87 @@
+/*  =========================================================================
+    secw_sshkey_and_login - Document parsers for SshKeyAndLogin document
+    Copyright (C) 2019 - 2023 Eaton
```

Comment:
new implem, (c) should start at the begining year of this implem, so 
Copyright (C) 2023 Eaton